app/js/util/i18n.js

```diff
@@ -8,6 +8,7 @@ translations.set(NO_POLICY_LISTED, {en: 'No policy', fr: 'Absence de politique'}
 translations.set(TOPICS, {en: 'Topics', fr: 'Sujets'});
 translations.set(MADE_BY, {en: 'Made by', fr: 'Fait par'});
 translations.set(SUGGEST_EDIT, {en: 'Suggest edit', fr: 'Suggérer une modification'});
+translations.set(TRANSLATION_BY, {en: 'Translated by', fr: 'Traduit par'});
```

Comment:
That is perfect :) Thanks a ton for the credits and also to have created ourpolitics.ca !